apps/site/lib/site_web/templates/trip_plan/_itinerary_tab.html.eex

```diff
@@ -1,17 +1,27 @@
-  <div class="m-trip-plan-results__itinerary-legs">
-    <%= @routes |> Enum.map(&icon_for_route/1) |> Enum.intersperse(fa "angle-right") %>
-  </div>
-  <div class="m-trip-plan-results__itinerary-length">
+  <h2>Itinerary <%= @index %></h2>
+  <div>
+    <%= if @itinerary_row_list.alerts? do %>
+      <span class="m-trip-plan-results__itinerary-alerts">
+        <%= fa "exclamation-triangle" %>
+      </span>
+    <% end %>
     <span class="m-trip-plan-results__itinerary-length-time">
       <%= Timex.format!(@itinerary.start, "{h12}:{m} {AM}") %> - <%= Timex.format!(@itinerary.stop, "{h12}:{m} {AM}") %>
     </span>
+    <span>
+      <% duration =  Timex.diff(@itinerary.stop, @itinerary.start, :minutes) %>
+      <%= if duration >= 60 do %>
+        <%= duration |> div(60) %> hr
+        <%= duration |> rem(60) %> min
+      <% else %>
+        <%= duration |> rem(60) %> min
+      <% end %>
```

Comment:
If you wanted, this section feels like a candidate for refactoring into a function in `TripPlanView`, maybe called `format_duration/2` or something along those lines. It would allow you to test that function and would make this template simpler and more readable.